docs/user-guide/ze-usage.md

```diff
@@ -0,0 +1,287 @@
+
+#### Delete a PDS member and PDS
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Open the profile and PDS containing the member.
+4. Right-click on the PDS member that you want to delete and select **Delete Member**.
+5. Confirm the deletion by clicking **Yes** on the drop-down menu.
+
+    **Note:** Alternatively, you can select 'No' to cancel the deletion.
+6. To delete a PDS, right-click the PDS and click **Delete PDS**, then confirm the deletion.
+
+    **Note:** You can delete a PDS before you delete its members.
+
+<img src="../images/ze/ZE-del-pds-member.gif" alt="Delete">
+
+#### View and access multiple profiles simultaneously
+
+1. Navigate to the explorer tree.
+2. Open the **DATA SETS** bar.
+3. Click the **Add Profile** button on the right of the **DATA SET** explorer bar.
+4. Select the profile that you want to add to the view as illustrated by the following screen.
+
+<img src="../images/ze/ZE-mult-profiles.gif" alt="Add Profile">
+
+### Work with USS Files
+
+You can use the following functionalities when interacting with USS files:
+
+* **View Unix System Services (USS) files**: You can view multiple USS files simultaneously.
+* **Rename USS files**: You can rename specified USS files.
+* **Download, edit, and upload existing USS files**: You can instantly pull USS files from the mainframe, edit them and upload back.
+* **Create and delete USS files and directories**: Enables you to easily create and delete both USS files and directories.
+* **View and access multiple profiles simultaneously**: Enables to work with USS files from multiple profiles.
+
+#### View Unix System Services (USS) files
+
+1. Navigate to the explorer tree.
+2. Open the **Unix System Services (USS)** bar.
+3. Select the profile that you want to filter.
+4. Click the **Search Unix System Services (USS) by Entering a Path** magnifying glass.
+5. From the drop-down, enter the path that you want as the root of your displayed tree.  
+  All child files and directories of that root file are displayed in the explorer tree.
+
+  **Note:** You will not be able to expand directories or files that you are not authorised for.
+
+   <img src="../images/ze/ZE-path.gif" alt="Enter Path">
+
+#### Refresh the list of files
+
+1. Navigate to the explorer tree.
+2. Click **Refresh All** button on the right of the **Unix System Services (USS)** explorer bar as illustrated by the following screen:
+
+<img src="../images/ze/ZE-refreshUSS.gif" alt="Refresh All">
+
+#### Rename USS files
+
+1. Navigate to the explorer tree.
+2. Open the **USS** bar.
+3. Select a USS file you want to rename.
+4. Right-click the USS file and select the **Rename USS file** option.
+5. Change the name of the USS file.
+
+#### Download, edit, and upload an existing file
+
+1. Click the file that you want to download.
+
+    **Note:** To view the files within a directory, click the directory to expand the tree.
+
+    The file displays in the text editor window of VSC.
+
```

Comment:
Is it odd here to have step, note, result, and then note? Can we somehow move one or both of the notes so that the procedure is no so broken up?

Reply:
I think the second note is more like a tip. I'd suggest to convert it with more appropriate text. Something like:

"Use syntax coloring to define file associations so that the suffix of your file is marked up."

On the other hand, though, I don't fully understand this note. Did the original author mean "syntax highlighter"? What is syntax coloring, anyway? @BrandonJenkins14 any ideas? This is not to say you are the original author :) but maybe you know where this is coming from.